loky/__init__.py

```diff
@@ -3,18 +3,11 @@
 :class:`ProcessPoolExecutor` and a function :func:`get_reusable_executor` which
 hide the pool management under the hood.
 """
-from concurrent.futures import (
-    Executor,
-    wait,
-    TimeoutError,
-    CancelledError,
-    as_completed,
-    ALL_COMPLETED,
-    FIRST_COMPLETED,
-    FIRST_EXCEPTION,
-    Future,
-)
```

Comment:
I prefer the original style of multi-line import statements. It's closer to the black formatting style that we might soon decide to adopt once we have finalized the drop of Python [2.7 - 3.6] versions.

Reply:
Ok, I reverted to original style and ordered statements like constants-classes-functions, like `isort` does.